Swift-Base/AppDelegate.swift

```diff
@@ -29,6 +29,12 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
             }
         #endif
         
+        #if DEBUG
+            // print documents directory and device ID
+            print("\n*******************************************\nDOCUMENTS\n\(NSSearchPathForDirectoriesInDomains(NSSearchPathDirectory.DocumentDirectory, NSSearchPathDomainMask.UserDomainMask, true)[0])\n*******************************************\n")
```

Comment:
@nikitafomin maybe we should make use our the function then? I think it would be correct.

func FSDebugPrint(items: Any...) {
    #if DEBUG
        print(items)
    #endif
}